### DISCUSSION TITLE
Education (level, process, progress, etc.)

### DISCUSSION THREAD

**Discussion by dexink** [ufbmi/OMRSE#256] [General]
There is a diversity of classes nearly equivalent to/highly similar to "education process", and therefore, in the light of the APRICOT Project's requisite subtasks, these classes may need to be reviewed. For additional efficiency, this may also require regard of the recently curated and digitized SEDOH patient data collection instruments, and possibly additional collections of relevant, digitized information, such as that included below.

Results of querying "education" in OMRSE:
<img width="1005" alt="Screenshot 2024-11-27 at 1 30 43 PM" src="https://github.com/user-attachments/assets/f17f38e7-35c3-4fbb-92ab-40cf8bd86acf">

See Spreadsheet for a Compendium of Social Risk Factor Codes from UCSF's Stephanie Chernitskiy:
[Compendium Social Risk Factors Codes 6.20.18.slsx](https://mcw0.sharepoint.com/:x:/s/DataScienceInstitute-Bill-Colbie/Eadi8xZ-upVDolieHmMW5BcBb5Kit1X0Zo2CMAVmHAXY5g?e=h74vOO) [Author: Stephanie Chernitskiy, University of California San Fransisco, E4A (Evidence 4 Action)]

Discussion is at next regular meeting is requested.

**Comment by mbrochhausen** [comment]
Is the statement that these classes are equivalent or highly similar to 'education process' based on the terminological similarity of the labels or is there a lack of differentiation in the definitions provided for those classes?

**Comment by hoganwr** [comment]
The classes all highlighted are in a taxonomy, and are differentiated clearly by definitions and their relationships to other classes. This is the design of the education modeling we did, it is not a bug or a problem in any way. 

The issue logged here is overly broad and somewhat ambiguous. If there are specific new classes needed then you should submit them as new term requests with a clear label, definition, and immediate superclass at a minimum.

**Comment by hoganwr** [comment]
I converted this issue to a discussion
